### DISCUSSION TITLE
Fast Textual Inversion in Runpod

### DISCUSSION THREAD

**Discussion by vesperialuna** [TheLastBen/fast-stable-diffusion#2246] [General]
hi ben,

i have tried this on runpod and the training was complete. but i was a little confused because the output file for this textual inversion training is a .ckpt file and not .pt file. am i missing something? thanks in advance.

**Comment by TheLastBen** [comment]
the output file is a lora safetensors file, the lora file system helps with the quality of the training

**Reply by vesperialuna** [reply]
i got it ben. it works like a charm. thank you!
